docs/joss_paper/paper.md

```diff
@@ -82,35 +82,35 @@ The loss functions described in the previous sections are often combined with a
 
 ## Grouped images
-Unpaired images can also be grouped in applications such as single-modality intra-subject registration, in which each subject has multiple images acquired, e.g. at different time points. Two dataset loaders are implemented for grouped image data, with- and without labels. Paired dataset loader is a special case of the grouped dataset loader. Prostate logitudinal MR is proposed to track the progression of prostate cancer patients at an early stage during the active surveilence programme [@yang2020longitudinal]. One DeepReg demo illustrates how the grouped dataset loader can be used for deep-learning based feature reigstration by aligning intra-patient organ masks.
+Unpaired images might also be grouped in applications such as _single-modality_ _intra-subject_ registration. In this case, each subject has multiple images acquired, e.g. at different time points. Two dataset loaders are implemented for grouped image data with- and without labels. Paired dataset loader is a special case of the grouped dataset loader. Prostate logitudinal MR registration is proposed to track the progression of prostate cancer patients at an early stage during the active surveilence programme [@yang2020longitudinal]. One DeepReg demo illustrates how the grouped dataset loader can be used for deep-learning-based feature reigstration by aligning intra-patient prostate gland masks.
 
 
 # Conclusion
-`DeepReg` facilitates a number of deep learning algorithms to train image registration networks, which provides a reference of basic functionality and performance to further research in fields such as medical image analysis and computer assisted intervention. It is in its current open-source format not only providing a tool for scientific research, but also welcoming contributions from wider communities above and beyond medical imaging, biomedical engineering and computer sicence.
+`DeepReg` facilitates a number of deep learning algorithms to train image registration networks, which provides a reference of basic functionality and performance to further research and education in fields such as medical image analysis and computer-assisted intervention. It is in its current open-source format that not only provides a tool for scientific research and higher education, but also welcomes contributions from wider communities above and beyond medical imaging, biomedical engineering and computer sicence.
```

Comment:
Repetition between "to further research and education" and "for scientific research and higher education"